scripts/generators/layer_dispatch_table_generator.py

```diff
@@ -52,5 +71,31 @@ def generate(self):
             out.extend([f'#endif //{command.protect}\n'] if command.protect else [])
         out.append('} VkLayerDispatchTable;\n')
 
+        out.append('''
+static inline void layer_init_device_dispatch_table(VkDevice device, VkLayerDispatchTable *table, PFN_vkGetDeviceProcAddr gdpa) {
```

Comment:
Should there be a `vk_` prefix? Or perhaps a `vul_` prefix for vulkan utility libraries.

Regardless I think these functions should be prefixed.

Reply:
vul_ makes sense.

Reply:
With a Vul_ prefix, should the file be renamed from `vk_layer_` to vul_`?

Reply:
`vul_dispatch_table.h` sounds good to me.